<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>easyWindow</class>
 <widget class="QDialog" name="easyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="optnThree">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>option 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="qstnLbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Question</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="optnOne">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>option 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="optnFour">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>option 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="optnTwo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>option 2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
